src/components/Tag/Tag.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { themePropType } from '../../util/shared-prop-types';
+import { textMega } from '../../styles/style-helpers';
+import DefaultCloseButton from '../CloseButton';
+
+const closeButtonStyles = ({ selected, theme }) => css`
+  label: tag__close-button;
+  margin-left: ${theme.spacings.kilo};
+  vertical-align: -1px;
+
+  > svg {
+    fill: ${selected && theme.colors.white};
+  }
+`;
+
+const CloseButton = styled(DefaultCloseButton)(closeButtonStyles);
+
+const tagStyles = ({ selected, theme }) => css`
+  label: tag;
+  padding: ${theme.spacings.bit} ${theme.spacings.kilo};
+  border-radius: ${theme.borderRadius.mega};
+  border: 1px solid ${theme.colors.n300};
+  background-color: ${selected ? theme.colors.b500 : theme.colors.n100};
+  color: ${selected && theme.colors.white};
```

Comment:
We've made it a convention to isolate styles that are related to a specific state (in this case `selected`). So this should be: 

```js
const tagBaseStyles = ({ theme }) => css`
  label: tag;
  padding: ${theme.spacings.bit} ${theme.spacings.kilo};
  border-radius: ${theme.borderRadius.mega};
  border: 1px solid ${theme.colors.n300};
  background-color: ${theme.colors.n100};
  ...
}`;

const tagSelectedStyles = ({ selected, theme }) => selected && css`
  label: tag--selected;
  background-color: ${theme.colors.b500};
  color: ${theme.colors.white};
}`;
```

This has the advantage of making the styles more composable and mostly, enables us to extract the correct class names during static style extraction (in this case `tag--selected`).

Reply:
Yeah, that makes sense. Let me change that